e2e/example_spec.js

```diff
@@ -0,0 +1,8 @@
+describe('angularjs homepage', () => {
+		it('should greet the named user',() => {
+				browser.get('http://www.angularjs.org');
```

Comment:
!fix
remove example spec and add test, that, say, checks *our code* that we have headline 'SmartHouse'